src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function (fog) {
```

Comment:
I think required arguments would have a different API from optional arguments. That is just my opinion, however.